extensions/amp-carousel/validator-amp-carousel.protoascii

```diff
@@ -68,3 +67,71 @@ tags: {  # <amp-carousel>
     supported_layouts: RESPONSIVE
   }
 }
+tags: {  # <amp-carousel> lightbox
+  tag_name: "AMP-CAROUSEL"
+  spec_name: "AMP-CAROUSEL[lightbox]"
```

Comment:
nit: I was told [in one of my validator PR reviews](https://github.com/ampproject/amphtml/pull/14533/files/f5fa7918825a30d583e0de61a2fac3dfa207b25c#r180571575) that it's convention for `spec_name`s to put a space between the tag name and the attribute `[`